draft-tuexen-tsvwg-rfc4960-errata.xml

```diff
@@ -733,6 +733,74 @@ partial_bytes_acked and cwnd in the congestion avoidance phase.</t>
 </section>
 </section>
 
+<section title='HEARTBEAT ACK and the association error counter'>
+<section title='Description of the Problem'>
+<t>Section 8.1 and Section 8.3 of <xref target="RFC4960"/>
+prescribe that the receiver of a HEARTBEAT ACK must reset the
+association overall error counter. In some circumstances, e.g. when
+a router discards DATA chunks but not HEARTBEAT chunks due to the
+larger size of the DATA chunk, it is better to not clear the
+association error counter by the HEARTBEAT ACK and reset it by the
+SACK only.</t>
+</section>
```

Comment:
I didn't get your comment. Indeed in PMTUD SCTP may send large HBs to check PMTU. In normal situation when HB much less than DATA it might ignore HB ACK if there is outstanding data. It is why the Description of the Problem clearly says "e.g. when a router discards DATA chunks but not HEARTBEAT chunks due to the larger size of the DATA chunk".